config/example_altimetry.json

```diff
@@ -0,0 +1,19 @@
+{
+	"type": "indexed",
+	"dimensionality": 1,
+	"chunks": [
+		"{'time':1000}"
+	],
```

Comment:
Chunks should just be a dict:
```json
"chunks": {
        'time':1000
}
```
Originally it was a list, but then has since been changed. (Dask accepts a list input for chunking, but that method doesn't allow naming of dimensions. I.e. (100, 100, 100 ). So this is where the list originally came from. )

Reply:
have modified.